sites/public/cypress.config.ts

```diff
@@ -35,7 +35,8 @@ export default defineConfig({
     specPattern: "cypress/e2e/**/*.{js,jsx,ts,tsx}",
     experimentalRunAllSpecs: true,
     env: {
-      showSeedsDesign: process.env.SHOW_NEW_SEEDS_DESIGNS,
+      showSeedsDesign: process.env.SHOW_NEW_SEEDS_DESIGNS === "TRUE",
+      runAccessibilityTests: process.env.RUN_ACCESSIBILITY_E2E_TESTS === "TRUE",
```

Comment:
question: this environment variable is not yet in the codebase of Doorway. It was created as part of https://github.com/bloom-housing/bloom/pull/5138. Should we hold off on adding it until we have it with the correct commit